### PR TITLE
Xorg: drop 32bit

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m [hunk 1]
```diff
@@ -28,13 +28,9 @@ file conf/11-x11-synaptics.fdi path=etc/hal/fdi/policy/10osvendor/11-x11-synapti
 
 file path=usr/include/xorg/synaptics-properties.h
 file path=usr/lib/$(MACH64)/pkgconfig/xorg-synaptics.pc
-file path=usr/lib/pkgconfig/xorg-synaptics.pc
 file path=usr/lib/xorg/$(MACH64)/synclient mode=0555
 file path=usr/lib/xorg/$(MACH64)/syndaemon mode=0555
 file path=usr/lib/xorg/modules/input/$(MACH64)/synaptics_drv.so
-file path=usr/lib/xorg/modules/input/synaptics_drv.so
```
Comment (pyhalov) [inline components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m:35]: on Linux synclient/syndaemon live in usr/bin, should we move at least synclient there?

File: components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m [hunk 2]
```diff
@@ -25,10 +25,7 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=etc/hal/fdi/policy/10osvendor/11-x11-vmmouse.fdi
 file path=usr/lib/$(MACH64)/xorg/modules/input/vmmouse_drv.so
 file path=usr/lib/hal/$(MACH64)/hal-probe-vmmouse
-file path=usr/lib/hal/hal-probe-vmmouse
 file path=usr/lib/xorg/$(MACH64)/vmmouse_detect
-file path=usr/lib/xorg/modules/input/vmmouse_drv.so
-file path=usr/lib/xorg/vmmouse_detect
```
Comment (pyhalov) [inline components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: should hal call these handlers? If so, do we still need 32-bit ones?

File: components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m (hunk 1 above)
Comment (alarcher) [reply components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m:35]: Sure I can do it.

File: components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m (hunk 2 above)
Comment (alarcher) [reply components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: Probably we need to adjust the path in hal.

Comment (pyhalov) [reply components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: Perhaps, it'll be ok with 64-bit binaries in 32-bit path.

Comment (pyhalov) [reply components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: but someone should look at hal-probe-vmmouse usage and test it :)

Comment (alarcher) [reply components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: /usr/lib/hal/amd64/hal-probe-vmmouse contains the full path to /usr/lib/xorg/amd64/vmmouse_detect so no problem, probably I should just move /usr/lib/hal/amd64/hal-probe-vmmouse to /usr/lib/hal/hal-probe-vmmouse

Comment (alarcher) [reply components/x11/xf86-input-vmmouse/manifests/sample-manifest.p5m:31]: Done

File: components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m (hunk 1 above)
Comment (alarcher) [reply components/x11/xf86-input-synaptics/xorg-input-synaptics.p5m:35]: Done

**Review by pyhalov — APPROVED** [review]

File: components/x11/tigervnc/Makefile [hunk 3]
```diff
@@ -17,6 +17,7 @@ BUILD_BITS= 64
 BUILD_STYLE= cmake
 
 include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
 
 COMPONENT_NAME= tigervnc
 COMPONENT_VERSION= 1.8.0
```
Comment (AndWac) [inline components/x11/tigervnc/Makefile:23]: Don't you want to change COMPONENT_REVISION?

File: components/x11/xf86-input-acecad/Makefile [hunk 4]
```diff
@@ -13,35 +13,20 @@
...
 
-BUILD_BITS = 32_and_64
+X11_CATEGORY=DRIVER
 include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
 
 COMPONENT_NAME= xf86-input-acecad
 COMPONENT_VERSION= 1.5.0
-COMPONENT_REVISIION= 2
+COMPONENT_REVISION= 2
```
Comment (AndWac) [inline components/x11/xf86-input-acecad/Makefile]: COMPONENT_REVISION unchanged?

File: components/x11/xf86-video-vboxvideo/Makefile [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+X11_CATEGORY=DRIVER
+include ../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/x11.mk
+
+COMPONENT_NAME= xf86-video-vboxvideo
+COMPONENT_VERSION= 1.0.0
+# Use version string from Xorg module
+IPS_COMPONENT_VERSION= 1.0.1
+COMPONENT_REVISION= 3
```
Comment (AndWac) [inline components/x11/xf86-video-vboxvideo/Makefile]: You didn't change the COMPONENT_REVISION here, either.

File: make-rules/shared-macros.mk [hunk 6]
```diff
@@ -578,7 +578,7 @@ export CCACHE := $(shell \
         fi; \
     fi)
 
-GCC_VERSION =	6
+GCC_VERSION =	7
```
Comment (AndWac) [inline make-rules/shared-macros.mk]: So, it's not only about dropping 32 Bit but also change to gcc 7?

**Review by AndWac — CHANGES_REQUESTED** [review]
- I think the COMPONENT_REVISIONS should be changed, too.

Changed.

- Is the change of gcc version already intended or just an overlook?

No just easier for me to rebuild until merge is decided.

File: components/x11/xf86-video-vboxvideo/Makefile (hunk 5 above)
Comment (alarcher) [reply components/x11/xf86-video-vboxvideo/Makefile]: I did but git auto merge decided otherwise..

File: components/x11/tigervnc/Makefile (hunk 3 above)
Comment (alarcher) [reply components/x11/tigervnc/Makefile:23]: Fixed, thank you!

File: components/x11/xf86-input-acecad/Makefile (hunk 4 above)
Comment (alarcher) [reply components/x11/xf86-input-acecad/Makefile]: Fixed, thank you!

File: components/x11/xf86-video-vboxvideo/Makefile (hunk 5 above)
Comment (alarcher) [reply components/x11/xf86-video-vboxvideo/Makefile]: Fixed, thank you!

File: make-rules/shared-macros.mk (hunk 6 above)
Comment (alarcher) [reply make-rules/shared-macros.mk]: It was a convenience for rebuilding here, I remove it before merge.

**Review by AndWac — APPROVED** [review]

**Comment by pyhalov** [comment]
please, saner commit messages ;)

**Pull request merged**